addon/authenticators/okta.js

```diff
@@ -1,10 +1,22 @@
 import { getOwner } from '@ember/application';
-import { inject as service } from '@ember-decorators/service';
-import { task } from 'ember-concurrency-decorators';
-import { timeout } from 'ember-concurrency';
+import { inject as service } from '@ember/service';
+import { task, timeout } from 'ember-concurrency';
```

Comment:
Can now just import the `@task` annotation directly from `ember-concurrency`.  Thank you very much!